### PR TITLE
chore: add flow alias to jsx flavor

### PR DESCRIPTION
**Pull request by ilevyor**
fixes: https://github.com/getgrit/gritql/issues/215

## Summary by CodeRabbit

- **New Features**
    - Added support for the "flow" and "flowComments" flavors in the `PatternLanguage` implementation.

### CODE REVIEW

File: crates/language/src/target_language.rs [hunk 1]
```diff
@@ -204,6 +204,7 @@ impl PatternLanguage {
         match name {
             "js" => match flavor {
                 Some("jsx") => Some(Self::Tsx),
+                Some("flow") => Some(Self::Tsx),
```
Comment (morgante) [inline crates/language/src/target_language.rs:207]: We should also support `flowComments` or remove it from the grammar: https://github.com/getgrit/tree-sitter-gritql/blob/5f4543610f49f7607b5ca54082da017837219a23/grammar.js#L64C1-L65C1

**Comment by romanlex** [comment]
how will this work?
If you now specify jsx in the Studio, then parsing still does not occur
https://app.grit.io/studio?key=MTd7mR3taj5k6i2sDL1mZ

**Comment by morgante** [comment]
JSX works fine in the studio: https://app.grit.io/studio?key=MTd7mR3taj5k6i2sDL1mZ

**Comment by romanlex** [comment]
@morgante 
another case
https://app.grit.io/studio?key=MTd7mR3taj5k6i2sDL1mZ

**Pull request merged**